src/main/java/edu/ucsb/nceas/mdqengine/scheduler/RequestReportJob.java

```diff
@@ -610,62 +621,273 @@ public void submitReportRequest(MultipartCNode cnNode, MultipartMNode mnNode, Bo
         SystemMetadata sysmeta = null;
         InputStream objectIS = null;
         InputStream runResultIS = null;
+        HashStore hashStore = null;
 
         Identifier pid = new Identifier();
         pid.setValue(pidStr);
 
         try {
-            if (isCN) {
-                sysmeta = cnNode.getSystemMetadata(session, pid);
-            } else {
-                sysmeta = mnNode.getSystemMetadata(session, pid);
-            }
-        } catch (NotAuthorized na) {
-            log.error("Not authorized to read sysmeta for pid: " + pid.getValue() + ", continuing with next pid...");
-            return;
+            hashStore = getHashStoreFromMetadigProps();
+
         } catch (Exception e) {
```

Comment:
we should catch and handle specific exceptions, not the catchall `Exception`. I've been trying to change this in the code base every time I touch old code